payment-intent/cs/payment-intent-manual.cs

```diff
@@ -13,5 +13,7 @@
     },
     Description = "Demo payment intent",
     Reference = "Order - 1",
-    CaptureMode = CaptureMode.Manual
+    CaptureMode = CaptureMode.Manual,
+    AutoExpireAt = DateTime.UtcNow.AddDays(7),//Replace with a future date of your choice.
```

Comment:
spaces please, after `,` and after `//`. We support maximum 7 days.

Reply:
Done